tests/test_smoke.py

```diff
@@ -0,0 +1,81 @@
+import yaml
+import subprocess
+import time
+import pytest
+
+
+def test_install():
+    with open("snap/snapcraft.yaml") as file:
+        snapcraft = yaml.safe_load(file)
+
+        subprocess.run(
+            f"sudo snap install ./{snapcraft['name']}_{snapcraft['version']}_amd64.snap --devmode".split(),
+            check=True,
+        )
+
+
+@pytest.mark.run(after="test_install")
+def test_all_apps():
+    with open("snap/snapcraft.yaml") as file:
+        snapcraft = yaml.safe_load(file)
+
+        override = {
+            "pgbackrest": "version",
+        }
```

Comment:
AFAIK, there are no pgbackrest here... 0_o

Reply:
Removed fluff overrides and skips.